### PR TITLE
Update free-programming-cheatsheets.md

### PR DESCRIPTION
**Pull request by Tnguyen-learning**
## What does this PR do?
Add resource(s) 

### Description
Add R color cheat sheet

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
please resolve conflicts.

**Comment by Tnguyen-learning** [comment]
@eshellman Could you please review my work?

**Comment by eshellman** [comment]
please resolve conflicts.

**Comment by Tnguyen-learning** [comment]
@eshellman I have resolved the conflicts.

**Review by shrn01 — APPROVED** [review]

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -339,6 +339,7 @@
 * [All RStudio cheatsheets resources](https://www.rstudio.com/resources/cheatsheets) - RStudio.com (HTML site with PDF links)
 * [Base R Cheatsheet](https://iqss.github.io/dss-workshops/R/Rintro/base-r-cheat-sheet.pdf) - Mhairi McNeill (PDF)
 * [Cheat Sheet for R and RStudio](https://www.ocf.berkeley.edu/~janastas/docs/RStudioCheatSheet.pdf) - L. Jason Anastasopoulos (PDF)
+* [R color cheatsheet](https://www.nceas.ucsb.edu/sites/default/files/2020-04/colorPaletteCheatsheet.pdf) - nceas.ucsb.edu (PDF)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:342]:
```suggestion
* [R color cheatsheet](https://www.nceas.ucsb.edu/sites/default/files/2020-04/colorPaletteCheatsheet.pdf) - Melanie Frazier (PDF)
```
Please add author names where appropriate.

**Pull request merged**